tools/base/fpgainfo/board.c

```diff
@@ -62,6 +62,30 @@ static platform_data platform_data_table[] = {
 	{ 0,      0,          NULL, NULL },
 };
 
+void *find_plugin(const char *libpath)
```

Comment:
I see some duplication with these "find" plugin functions.
Is it worthwhile/feasible to refactor into one function?

Reply:
Not really, since they're all in different libraries.  Defining them inline in a header might be a way, but we don't have any precedent for using inline fn's that way.